From 48aba7f3c838cd88be98c01091de60793a763609 Mon Sep 17 00:00:00 2001
From: Felipe Balbi <balbi@ti.com>
Date: Thu, 13 Oct 2011 11:35:02 +0300
Subject: [PATCH 0363/1092] cbus: mark dependencies for ARCH_OMAP

This is just a temporary commit. Ideally, all
these drivers wouldn't depend on a particular
architecture, but some of those Retu/Tahvo
children are using headers under <plat>, so
we must add this dependency until that's sorted
out.

Signed-off-by: Felipe Balbi <balbi@ti.com>
Signed-off-by: Tony Lindgren <tony@atomide.com>
---
 drivers/cbus/Kconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/cbus/Kconfig b/drivers/cbus/Kconfig
index 938d244..41d96e7 100644
--- a/drivers/cbus/Kconfig
+++ b/drivers/cbus/Kconfig
@@ -24,6 +24,7 @@ if CBUS_TAHVO
 
 config CBUS_TAHVO_USB
 	depends on USB
+	depends on ARCH_OMAP
 	select USB_OTG_UTILS
 	tristate "Support for Tahvo USB transceiver"
 	---help---
@@ -58,6 +59,7 @@ config CBUS_RETU_POWERBUTTON
 
 config CBUS_RETU_RTC
 	depends on RTC_CLASS
+	depends on ARCH_OMAP
 	tristate "Support for Retu pseudo-RTC"
 	---help---
 	  Say Y here if you want support for the device that alleges to be an
@@ -65,6 +67,7 @@ config CBUS_RETU_RTC
 
 config CBUS_RETU_WDT
 	depends on SYSFS && WATCHDOG
+	depends on ARCH_OMAP
 	tristate "Support for Retu watchdog timer"
 	---help---
 	  Say Y here if you want support for the watchdog in Retu. This will
-- 
1.7.11.2

